Adjust Fuzzy Matching for backdrop-filter Test to Reduce Rendering Variability

Jean-saintfleur opened the pull request:
#### f6d3b0c1b98a174719f55be1694ee7ab61331c09
<pre>
Adjust Fuzzy Matching for backdrop-filter Test to Reduce Rendering Variability

Adjusts fuzzy matching to prevent false failures caused by minor pixel differences in backdrop-filter rendering.

* css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter.html:
* css/filter-effects/backdrop-filter-backdrop-root-opacity.html:
* css/filter-effects/backdrop-filter-clip-rect-zoom.html:
* css/filter-effects/css-filters-animation-brightness.html:
</pre>

Ahmad-S792 commented on css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter.html:
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
+<meta name="fuzzy" content="0-1;0-2696">
```
Can push it after `<meta name="match">` for all.

Jean-saintfleur committed:
Adjust Fuzzy Matching for backdrop-filter Test to Reduce Rendering Variability

Adjusts fuzzy matching to prevent false failures caused by minor pixel differences in backdrop-filter rendering.

* css/filter-effects/backdrop-filter-backdrop-root-backdrop-filter.html:
* css/filter-effects/backdrop-filter-backdrop-root-opacity.html:
* css/filter-effects/backdrop-filter-clip-rect-zoom.html:
* css/filter-effects/css-filters-animation-brightness.html:

jnz555 reviewed (commented):
https://www.paypal.com/ncp/payment/CE2SBU46UBLGU[](url)
